src/viewer.js

```diff
@@ -1944,7 +1944,28 @@ $.extend( $.Viewer.prototype, $.EventSource.prototype, $.ControlDock.prototype,
         this.raiseEvent( 'clear-overlay', {} );
         return this;
     },
+    
+     /**
+     * Finds an overlay identified by the reference element or element id
+     * and returns it as an object, return null if not found.
+     * @method
+     * @param {Element|String} element - A reference to the element or an
+     *      element id which represent the ovelay content to be removed.
```

Comment:
No, it’s intended use is simpler access to overlays in user applications

-- 
Alexey Tikhonov


On 23 February 2016 at 00:20:25, Grant Echols (notifications@github.com) wrote:

In src/viewer.js:

> @@ -1944,7 +1944,28 @@ $.extend( $.Viewer.prototype, $.EventSource.prototype, $.ControlDock.prototype,
>          this.raiseEvent( 'clear-overlay', {} );
>          return this;
>      },
> +     
> +     /**
> +     * Finds an overlay identified by the reference element or element id
> +     * and returns it as an object, return null if not found.
> +     * @method
> +     * @param {Element|String} element - A reference to the element or an
> +     *      element id which represent the ovelay content to be removed.
Does this method only get used when you are going to remove an overlay? The comment seems to suggest it does but I can think of other reasons it could be useful.

—
Reply to this email directly or view it on GitHub.